test/adapters/redis_client_test.rb

```diff
@@ -257,22 +264,21 @@ def test_timeout_changes_when_half_open_and_configured_with_writes_and_disconnec
     client.close
 
     time_circuit_half_open = ERROR_TIMEOUT + 1
-    time_travel(time_circuit_half_open) do
-      assert_redis_timeout_in_delta(expected_timeout: half_open_resource_timeout) do
+    assert_redis_timeout_in_delta(expected_timeout: half_open_resource_timeout) do
+      time_travel(time_circuit_half_open) do
```

Comment:
Changed order, because `assert_redis_timeout_in_delta` uses `Benchmark.measure` that uses `Process.clock_gettime(Process::CLOCK_MONOTONIC)`.